static/js/sefaria/sefaria.js

```diff
@@ -2206,7 +2206,20 @@ _media: {},
       let refToCheck = Array.isArray(ref) ? ref[0] : ref;
       const parsedRef = Sefaria.parseRef(refToCheck);
       const book = Sefaria.index(parsedRef.index);
-      return this.isCommentaryWithBaseText(book);
+      if (!this.isCommentaryWithBaseText(book)) {
+          return false;
+      }
+
+      // by this point, we know the book is in the right form, but we still need to check that the ref is in the right form
```

Comment:
could this logic be useful elsewhere as a helper function? It would also be cleaner to have a function name describing what this code is doing (isValideRef or something of the like)